api/environments/migrations/0029_add_default_value_for_use_mv_v2_evaluation.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.16 on 2023-02-03 05:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("environments", "0028_add_use_mv_v2_evaluation"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Can we combine this with the previous migration? I guess we want a different default to be used on existing environments versus new ones, but we could still just have all the operations in the same migration, right? Or is there a way we can use the `preserve_default` argument? (I think not but worth checking).

Reply:
Yeah, it's not possible to do in one operation